CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# [2.0.0-alpha.1](https://github.com/gravitee-io/gravitee-kubernetes/compare/1.0.0...2.0.0-alpha.1) (2022-11-08)
```

Comment:
just because we wondered about making a specific changelog for alpha to avoid having it here. but 🤷‍♂️